cmd/serve/docker/driver.go

```diff
@@ -40,7 +40,7 @@ type Driver struct {
 // NewDriver makes a new docker driver
 func NewDriver(ctx context.Context, root string, mntOpt *mountlib.Options, vfsOpt *vfscommon.Options, dummy, forgetState bool) (*Driver, error) {
 	// setup directories
-	cacheDir, err := filepath.Abs(config.CacheDir)
+	cacheDir, err := filepath.Abs(config.GetCacheDir())
```

Comment:
This no longer needs a `filepath.Abs` does it?

Reply:
Correct. Good catch.